src/wasm-lib/kcl/src/std/math.rs

```diff
@@ -23,13 +23,15 @@ pub async fn rem(_exec_state: &mut ExecState, args: Args) -> Result<KclValue, Kc
 /// If `num` is negative, the result will be too.
 ///
 /// ```no_run
-/// assertEqual(rem(7, 4),  3, 0.01, "remainder is 3")
-/// assertEqual(rem(-7, 4), -3, 0.01, "remainder is 3")
-/// assertEqual(rem(7, -4), 3, 0.01, "remainder is 3")
+/// assertEqual(rem(7, divisor: 4),  3, 0.01, "remainder is 3")
+/// assertEqual(rem(-7, divisor: 4), -3, 0.01, "remainder is 3")
+/// assertEqual(rem(7, divisor: -4), 3, 0.01, "remainder is 3")
```

Comment:
If we make this change, it forces users to use keyword arguments. So doesn't the TS need updates to handle the new node type?

Reply:
Yes, correct -- doing that now.